### PR TITLE
Fixes #9569 - do not lay the gutterball.conf file

### PR DESCRIPTION
**Pull request by dustints**

### CODE REVIEW

**Comment by dustints** [comment]
couldn't get successful scratch build without removing these lines.

**Comment by dustints** [comment]
@komidore64 can you help review please?

File: rubygem-foreman_gutterball.spec [hunk 1]
```diff
@@ -69,12 +69,6 @@ gem install --local --install-dir .%{gem_dir} --force %{SOURCE0} --no-rdoc --no-
 %{__cp} --archive .%{gem_dir}/gems %{buildroot}%{gem_dir}/
 %{__cp} --archive .%{gem_dir}/specifications %{buildroot}%{gem_dir}/
 
-%{__install} --directory %{buildroot}%{foreman_bundlerd_dir}
-
-%foreman_bundlerd_file
```
Comment (komidore64) [inline rubygem-foreman_gutterball.spec]: lines `72-74` are still needed. this is how foreman knows to load the plugin.

File: rubygem-foreman_gutterball.spec [hunk 2]
```diff
@@ -89,7 +83,6 @@ gem install --local --install-dir .%{gem_dir} --force %{SOURCE0} --no-rdoc --no-
 %{gem_instdir}/test/
 %{gem_spec}
 %{gem_cache}
-%{foreman_bundlerd_plugin}
```
Comment (komidore64) [inline rubygem-foreman_gutterball.spec]: this is still needed too

**Pull request closed without merging**